app/models/miq_server/server_monitor.rb

```diff
@@ -16,25 +16,25 @@ def mark_as_not_responding(seconds = miq_server_time_threshold)
   end
 
   def make_master_server(last_master)
-    _log.info "Master server has #{last_master.nil? ? "not been set" : "died"}.  Attempting takeover as new master server."
+    _log.info "Master server has #{last_master.nil? ? "not been set" : "died, #{last_master.name}"}.  Attempting takeover as new master server, #{name}."
     parent = MiqRegion.my_region
     parent.lock do
       all_servers = parent.miq_servers
 
       _log.debug "Double checking that nothing has changed"
       master = all_servers.detect(&:is_master?)
       if (last_master.nil? && !master.nil?) || (!last_master.nil? && !master.nil? && last_master.id != master.id)
-        _log.info "Aborting master server takeover as another server has taken control first."
+        _log.info "Aborting master server takeover as another server, #{master.name}, has taken control first."
         return nil
       end
 
-      _log.debug "Setting this server as master server"
       all_servers.each do |s|
+      _log.debug "Setting this server, #{name}, as master server"
```

Comment:
I think it wouldn't hurt to always log this one too. It'll give us some more insight as to what's going the next time we have to chase a bug in this code.